subprojects/testing-jvm/src/integTest/resources/org/gradle/testing/junit/JUnitPlatformIgnoreClassIntegrationSpec/canHandleClassLevelIgnoredTests/build.gradle

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
```

Comment:
Additional new lines

Reply:
Removed.

Reply:
I still see 3 extra lines in there.

Reply:
This part has been moved into test class body instead of isolated resource files.